### ISSUE TITLE
blood splatter while fire some modded arrow

### ISSUE THREAD

**Issue by Biviho** [CreativeMD/EnhancedVisuals#38]
using arrow from better with mods, ice and fire and some other result in a splatter effect on the screen.

**Comment by CreativeMD** [comment]
Is that an issue or a suggestion? I'm a little confused.

**Comment by Biviho** [comment]
an issue, firing an arrow from those mod cause multiple blood splatter effect on screen, like you are "hitting yourself" with the arrow, after 5 arrow the screen become almost completely red.
if you don't have this problem it may be some other mod that cause this strange interaction only with some modded arrow, i've seen this in a costum modpack, i've not yet tested the issue with only those mod and enhanced visual

**Comment by CreativeMD** [comment]
This will be changed once the rewrite is done. At the moment this cannot be fixed because there is no way for EnhancedVisuals to get the actual damage the player received.

**Comment by CreativeMD** [comment]
This should be fixed in v1.4.0, if not open a new issue.
